bin/swift4-test.sh

```diff
@@ -27,7 +27,7 @@ fi
 
 # if you've executed sbt assembly previously it will use that instead.
 export JAVA_OPTS="${JAVA_OPTS} -Xmx1024M -DloggerPath=conf/log4j.properties"
-ags="generate -t modules/openapi-generator/src/main/resources/swift4 -i modules/openapi-generator/src/test/resources/2_0/swift/swift4Test.json -g swift4 -c ./bin/swift4-test.json -o samples/client/test/swift4/default --generate-alias-as-model $@"
+ags="generate -t modules/openapi-generator/src/main/resources/swift4 -i modules/openapi-generator/src/test/resources/2_0/swift4Test.json -g swift4 -c ./bin/swift4-test.json -o samples/client/test/swift4/default --generate-alias-as-model $@"
```

Comment:
Can you please explain why are you using 
`modules/openapi-generator/src/test/resources/2_0/swift4Test.json`
instead of 
`modules/openapi-generator/src/test/resources/2_0/swift/swift4Test.json`
?
I'm asking because I'm not sure which file to use when 🙂

Reply:
`modules/openapi-generator/src/test/resources/2_0/swift/swift4Test.json` doesn't exist so I update the path to use the right one.